src/_pytest/main.py

```diff
@@ -643,10 +713,11 @@ def _recurse(self, dirpath):
         return True
 
     @staticmethod
-    def _visit_filter(f):
-        return f.check(file=1)
+    def _visit_filter(f: py.path.local) -> bool:
+        # TODO: Remove type: ignore once `py` is typed.
+        return f.check(file=1)  # type: ignore
```

Comment:
Better would be to add the [`warn_unused_ignores`](https://mypy.readthedocs.io/en/stable/config_file.html#configuring-warnings) option to our mypy config, so that the tools remind us :smile:

Reply:
@Zac-HD 
I think the plan is to run this manually from time to time - with that option in the config it might vary between mypy versions etc.
But it makes sense to not use a "TODO" comment then, of course.  (and I prefer having the error code with the ignore btw (created https://github.com/pytest-dev/pytest/pull/6587)).